### PR TITLE
Add example of extra args on README

### PR DESCRIPTION
**Pull request by pipo02mix**
This PR:

---

## Checklist

- [ ] Added a CHANGELOG entry

## Testing

The instance of external-dns installed as part of Giant Swarm platform releases watches services in the `kube-system` namespace with annotations `giantswarm.io/external-dns=managed` and `external-dns.alpha.kubernetes.io/hostname` matching the clusters base domain. (You can find this in the deployments args `--domain-filter` value)

You can take this example `Service`, apply it to your cluster. Change the `external-dns.alpha.kubernetes.io/hostname` annotation to match your clusters base domain.

then:

- Check external-dns logs for lines like `Desired change: CREATE test.your.configured.domain.gigantic.io CNAME`
- Try to resolve the domain (`https://www.dnstester.net/`)

```yaml
apiVersion: v1
kind: Service
metadata:
  annotations:
    external-dns.alpha.kubernetes.io/hostname: test.your.configured.domain.gigantic.io
    external-dns.alpha.kubernetes.io/ttl: "60"
    giantswarm.io/external-dns: managed
  name: test-external-dns
  namespace: kube-system
spec:
  type: ExternalName
  externalName: www.giantswarm.io
```

For testing upgrades:

- Create the service and check for creation
- Upgrade
- Delete the service and check for deletion

### Default app on AWS releases

- [ ] Fresh install works
- [ ] Upgrade works

### Default app on Azure releases

- [ ] Fresh install works
- [ ] Upgrade works

### Optional app (KVM)

- [ ] Fresh install works
- [ ] Upgrade works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ This is an example of a values file you could upload using our web interface. It
 - API access is internal and therefore authentication is provided by KIAM.
 - Only public Hosted Zones should be updated.
 - Only Ingress resources in the namespace `web-app` should be reconciled.
+- Only Hosted Zone `Z262CGXUQ3M97` will be modified
```
Comment (webwurst) [inline README.md:42]:
```suggestion
- Only Hosted Zone `Z262CGXUQ3M97` will be modified.
```
Just because the others end with a dot.

**Review by webwurst — APPROVED** [review]

**Comment by webwurst** [comment]
Question:
> Only public Hosted Zones should be updated.
Is that really addressed in the example?

File: README.md [hunk 2]
```diff
@@ -56,6 +57,8 @@ externalDNS:
     txtPrefix: 'webapp'
   sources:
     - ingress
+  extraArgs:
+    - "--zone-id-filter: Z262CGXUQ3M97"
```
Comment (ubergesundheit) [inline README.md:61]:
```suggestion
    - "--zone-id-filter=Z262CGXUQ3M97"
```

File: README.md [hunk 3]
```diff
@@ -39,6 +39,7 @@ This is an example of a values file you could upload using our web interface. It
 - API access is internal and therefore authentication is provided by KIAM.
 - Only public Hosted Zones should be updated.
```
Comment (pipo02mix) [inline README.md:40]:
```suggestion
```

**Comment by pipo02mix** [comment]
```Question:

Only public Hosted Zones should be updated.

Is that really addressed in the example?```

good point @webwurst, I have remove it

**Review by webwurst — APPROVED** [review]

**Pull request merged**